lib/Mojo/Reactor/EV.pm

```diff
@@ -44,9 +44,9 @@ sub watch {
     my $cb = sub {
       my ($w, $revents) = @_;
       $self->_try('I/O watcher', $self->{io}{$fd}{cb}, 0)
-        if EV::READ & $revents;
+        if EV::READ & $revents && $self->{io}{$fd} && $self->{io}{$fd}{cb};
```

Comment:
should this be 
```suggestion
        if EV::READ & $revents && defined($self->{io}{$fd}{cb});
```

Reply:
Then perl will create `$self->{io}{$fd}` automatically if there is no such one: 
```
perl -e 'use strict; use warnings; my $a = {}; print "ok" if defined($a->{test}{cb}); use Data::Dumper; print Dumper($a);'
$VAR1 = {
          'test' => {}
        };

```